test/test_smoke.py

```diff
@@ -166,6 +167,21 @@ def test_image_boots(image_type):
         assert "hello" in output
 
 
+@pytest.mark.skipif(platform.system() != "Linux", reason="boot test only runs on linux right now")
+@pytest.mark.parametrize("image_type", CLOUD_IMAGE_TYPES, indirect=["image_type"])
+def test_image_boots_cloud(image_type):
+    if image_type.img_type == "ami":
```

Comment:
I wonder if it would make sense to just call this `test_image_boots_ec2` or `test_ami_boots_in_cloud` and hard code `@pytest.mark.parametrize("image_type", ["ami"], indirect=["image_type"])` - mostly to the `if image_type.img_type == ` checks, if all tests will look slightly different anyway we can as well make them different ?

Reply:
Yeah, fair.  It'll just end up being a big `switch` (or `if/else`s) in the end with no commonalities.
Also, we can probably enable it for macOS too (not that I'm looking at the pytest.marks again.

Reply:
Done.